fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/ApiRequestImpl.java

```diff
@@ -416,7 +314,7 @@ protected void validateMetrics(Set<LogicalMetric> logicalMetrics, LogicalTable t
             Granularity granularity,
             DateTimeFormatter dateTimeFormatter
     ) throws BadApiRequestException {
-        return generateIntervals(new DateTime(), apiIntervalQuery, granularity, dateTimeFormatter);
+        return IntervalBinders.generateIntervals(new DateTime(), apiIntervalQuery, granularity, dateTimeFormatter);
```

Comment:
How do you feel about aligning this syntax to the generators above?

Reply:
I think this will be clearer when IntervalBinders is renamed to IntervalUtils.